HttpFs/HttpFs.fs

```diff
@@ -142,8 +142,8 @@ module Logging =
 
   let NoopLogger =
     { new Logger with
-        member x.logVerbose evaluate = Alt.always (Promise.Now.withValue ())
-        member x.log message = Alt.always (Promise.Now.withValue ())
+        member x.logVerbose evaluate = Alt.always (Promise ())
+        member x.log message = Alt.always (Promise ())
```

Comment:
That actually has the wrong semantics. :(

See [here](https://github.com/Hopac/Hopac/issues/113).

@haf Yeah, I'm now convinced that we should have a clearer API for creating promises.